packages/webapp/components/layouts/AccountLayout/SidebarNav.tsx

```diff
@@ -50,6 +61,15 @@ function SidebarNav({
     };
   }, [router]);
 
+  const onItemClick: Partial<FunctionRecord<AccountPage>> = {
```

Comment:
Not sure we should track it here. This leads to the notification settings if I understand correctly

Reply:
Ah, correct. This was a mistake. Let me fix this quickly.

Reply:
Ah, correct. This was a mistake. Let me fix this quickly. I saw the possible values to have `Sidebar` so I mistakenly had added this too.